cg/store/filters/status_illumina_sequencing_run_filters.py

```diff
@@ -0,0 +1,31 @@
+"""Filters for the Illumina Sequencing Run objects."""
+
+from enum import Enum
+from typing import Callable
+
+from sqlalchemy.orm import Query
+
+from cg.store.models import IlluminaFlowCell
+
+
+def filter_by_run_internal_id(runs: Query, run_id: str, **kwargs) -> Query:
```

Comment:
This is not really the "run_id" but really the flow_cell name. I don't think we will be able to completely abstract it to run id. Since the flow_cell_name is really an attribute of the cartridge and a smaller part of the full sequencing_run_id (i.e. date, flow cell name etc.)

Reply:
fixed to `device_internal_id`